en/deployment/articles/03-installing-a-search-engine/02-elasticsearch/07-security.markdown

```diff
@@ -169,27 +175,47 @@ configuration (`elasticsearch.yml`) used in this guide (with the Elasticsearch 6
 example commented out):
 
 ```yaml
-# For Elasticsearch 7
+# For Elasticsearch 7.3/7.4
 cluster.name: LiferayElasticsearchCluster
+
+# X-Pack Security
 xpack.security.enabled: true
+
+## TLS/SSL settings for Transport layer
 xpack.security.transport.ssl.enabled: true
-xpack.security.transport.ssl.verification_mode: certificate
-xpack.security.transport.ssl.key: /path/to/[Elasticsearch Home]/config/localhost.crt
-xpack.security.transport.ssl.certificate: /path/to/[Elasticsearch Home]/config/localhost.key
-xpack.security.transport.ssl.certificate_authorities: ["/path/to/ca.crt"]
+xpack.security.transport.ssl.verification_mode: certificate 
+xpack.security.transport.ssl.key: /path/to/localhost.key
+xpack.security.transport.ssl.certificate: /path/to/localhost.crt
+xpack.security.transport.ssl.certificate_authorities : ["/path/to/ca.crt"]
```

Comment:
@lipusz we intentionally use a path inside ES Home/config, because this has been necessary in the past. Did you verify that it is no longer necessary?

Reply:
See below.